src/services/pcn-router/src/PortsSecondaryip.cpp

```diff
@@ -1,118 +1,55 @@
-/*
- * Copyright 2018 The Polycube Authors
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
+/**
+* router API generated from router.yang
+*
+* NOTE: This file is auto generated by polycube-codegen
+* https://github.com/polycube-network/polycube-codegen
+*/
```

Comment:
Is this file entirely generated by swagger without any modification? If we made some changes, it is better to add some comments here.

Reply:
Yes, it is.
The header changes because the router used a previous version of the code generation tool; in the latest version, we changed also the autogenerated header.